lib/ui/screen/login_screen.dart

```diff
@@ -85,121 +86,126 @@ class _LoginScreenState extends State<LoginScreen> {
               padding: const EdgeInsets.all(32),
               child: Form(
                 key: _formKey,
-                child: Column(
-                  crossAxisAlignment: CrossAxisAlignment.start,
-                  children: <Widget>[
-                    Material(
-                      elevation: 2,
-                      borderRadius: const BorderRadius.all(Radius.circular(32)),
-                      child: TextFormField(
-                        controller: _accountControl,
-                        cursorColor: Colors.blue[800],
-                        textInputAction: TextInputAction.done,
-                        focusNode: _accountFocus,
-                        onEditingComplete: () {
-                          _accountFocus.unfocus();
-                          FocusScope.of(context).requestFocus(_passwordFocus);
-                        },
-                        validator: (value) => _validatorAccount(value),
-                        decoration: InputDecoration(
-                          hintText: R.current.account,
-                          errorStyle: const TextStyle(
-                            height: 0,
-                            fontSize: 0,
+                child: AutofillGroup(
+                  child: Column(
+                    crossAxisAlignment: CrossAxisAlignment.start,
+                    children: <Widget>[
+                      Material(
+                        elevation: 2,
+                        borderRadius: const BorderRadius.all(Radius.circular(32)),
```

Comment:
Suggest changing to `BorderRadius.circular`.

```suggestion
                        borderRadius: const BorderRadius.circular(32),
```